kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyReconciler.java

```diff
@@ -122,9 +123,13 @@ public UpdateControl<KafkaProxy> reconcile(KafkaProxy primary,
     public ErrorStatusUpdateControl<KafkaProxy> updateErrorStatus(KafkaProxy proxy,
                                                                   Context<KafkaProxy> context,
                                                                   Exception e) {
+        if (e instanceof StaleReferentStatusException || e instanceof OperatorException && e.getCause() instanceof StaleReferentStatusException) {
+            LOGGER.info("Completed reconciliation of {}/{} with transient error {}", namespace(proxy), name(proxy), e.toString());
```

Comment:
It's a bit odd to call it an "transient error" when we it's expected. And we're not logging it at error (TBH I think even `info` is probably too loud).

Reply:
dropped it to debug and reworded